packages/ui/src/utils/update-ids.ts

```diff
@@ -0,0 +1,22 @@
+import { getCamelRandomId } from '../camel-utils/camel-random-id';
```

Comment:
I think this might be useful for other scenarios as well, perhaps moving it to the `utils` folder might be convenient.

Reply:
My bad, I guess I was confused with another file then :+1: